libs/remix-ui/solidity-compile-details/src/lib/solidity-compile-details.tsx

```diff
@@ -1,6 +1,8 @@
 import { CopyToClipboard } from '@remix-ui/clipboard'
 import { CustomTooltip } from '@remix-ui/helper'
 import { TreeView, TreeViewItem } from '@remix-ui/tree-view'
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import { ContractPropertyName } from 'libs/remix-ui/solidity-compiler/src/lib/types'
```

Comment:
please check but I think this isn't the right to write this import. You hae to use the module name (`solidity-compiler`)

Reply:
You are right. Its done.